sdk/cosmos/sdk/src/test/java/com/azure/data/cosmos/sync/CosmosSyncDatabaseTest.java

```diff
@@ -0,0 +1,202 @@
+    public void beforeClass() {
+        client = clientBuilder().buildSyncClient();
+        createdDatabase = createSyncDatabase(client, preExistingDatabaseId);
+    }
+
+    @AfterClass(groups = {"emulator"}, timeOut = SHUTDOWN_TIMEOUT, alwaysRun = true)
+    public void afterClass() {
+        safeDeleteSyncDatabase(createdDatabase);
+        for (String dbId : databases) {
+            safeDeleteSyncDatabase(client.getDatabase(dbId));
+        }
+        safeCloseSyncClient(client);
+    }
+
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT)
+    public void createDatabase() throws CosmosClientException {
+        CosmosDatabaseProperties databaseDefinition = new CosmosDatabaseProperties(CosmosDatabaseForTest.generateId());
+        databases.add(databaseDefinition.id());
+
+        CosmosSyncDatabaseResponse createResponse = client.createDatabase(databaseDefinition,
+                new CosmosDatabaseRequestOptions());
+
+        validateDatabaseResponse(databaseDefinition, createResponse);
+    }
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT)
+    public void createDatabase2() throws Exception {
+        CosmosDatabaseProperties databaseDefinition = new CosmosDatabaseProperties(CosmosDatabaseForTest.generateId());
+        databases.add(databaseDefinition.id());
+        CosmosDatabaseProperties databaseProperties = new CosmosDatabaseProperties(databaseDefinition.id());
+
+        CosmosSyncDatabaseResponse createResponse = client.createDatabase(databaseProperties);
+        validateDatabaseResponse(databaseDefinition, createResponse);
+    }
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT)
+    public void createDatabase3() throws Exception {
+        CosmosDatabaseProperties databaseDefinition = new CosmosDatabaseProperties(CosmosDatabaseForTest.generateId());
+
+        CosmosSyncDatabaseResponse createResponse = client.createDatabase(databaseDefinition.id());
+        validateDatabaseResponse(databaseDefinition, createResponse);
+    }
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT)
+    public void createDatabase4() throws Exception {
+        CosmosDatabaseProperties databaseDefinition = new CosmosDatabaseProperties(CosmosDatabaseForTest.generateId());
+        CosmosDatabaseProperties databaseProperties = new CosmosDatabaseProperties(databaseDefinition.id());
+        CosmosDatabaseRequestOptions requestOptions = new CosmosDatabaseRequestOptions();
+        int throughput = 400;
+        try {
+            CosmosSyncDatabaseResponse createResponse = client.createDatabase(databaseProperties, throughput, requestOptions);
+            validateDatabaseResponse(databaseDefinition, createResponse);
+        } catch (CosmosClientException ex) {
+            assertThat(ex.statusCode()).isEqualTo(HttpConstants.StatusCodes.FORBIDDEN);
+        }
+    }
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT)
+    public void createDatabase5() throws Exception {
```

Comment:
The same of the test methods here.
The test method name should speak of the nature of the test. what's the different between createDatabase4 vs createDatabase5?

Please add more context to the test method name.

Reply:
Renamed